### DISCUSSION TITLE
allow adjusting the mania background's transparency

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#18043] [Ideas]
the mania background that the notes fall on is completely solid black

i would like a skin setting that adjusts the transparency of the mania background where 0% doesn't show the background and 100% displays the background solidly

having a lower mania background transparency setting allows you to see the background, video, and/or storyboard while having a higher mania background transparency setting lets you focus on the gameplay

**Comment by peppy** [comment]
This should definitely be an easy one once we migrate per-ruleset playfields to the new skin editor system. I had a preliminary look at what would be involved to make this happen, but there is a considerable amount of work to be done. It's a bit of an all-or-nothing migration, I'd estimate about 5-10 hours for mania alone.
